display_png.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from PIL import Image
+
+logger = logging.getLogger(__name__)
+
+
+class Display:
+    def __init__(self, _cfg):
+        self.logger = logging.getLogger(__name__)
+        # FIXME: height from config file?
+        self.filename = 'display.png'
+        self.height = 600
+        self.width = 800
```

Comment:
Yes please, let's move width and height to config parameters.
The default display class gets the width and height of the display automatically, so the config parameters should be optional. Width 800 and height 600 seems like sensible defaults.